sorc/global_cycle.fd/read_write_data.f90

```diff
@@ -114,17 +115,21 @@ MODULE READ_WRITE_DATA
    !! @param[in] slcfcs Liquid portion of volumetric soil moisture.
    !! @param[in] smcfcs Total volumetric soil moisture.
    !! @param[in] stcfcs Soil temperature.
-   !!
+   !! @param[in] stc_inc_tmp Soil temperature increments on the cubed-sphere tiles
+   !! @param[in] slc_inc_tmp Liquid soil moisture increments on the cubed-sphere tiles
    !! @author George Gayno NOAA/EMC
-
+   !! @author Yuan Xue: gsi-based soil increments are interpolated onto the
```

Comment:
Unless you're using it, I don't think we need this (we can diff the anal and back to get the increment). I think my comment about doing this was out-dated. I suppose we may as well leave it in, in case it's useful in the future?

Reply:
Will leave it for now per discussion.